hooks/zfs/zfs

```diff
@@ -0,0 +1,9 @@
+# vim: set ft=sh:
+# shellcheck shell-sh
+
+copy_file "${zfs_cache#*=}" /etc/zfs/zpool.cache 600 0
+
+copy_module zfs
+
+copy_binary zfs
+copy_binary zpool
```

Comment:
For native encryption we will need the zfs binary :X

Reply:
Yeah. Could 'zfs load-key' do automatically no-op if dataset is unencrypted? That would allow us to not write code which checks if dataset is encryption or not.

Reply:
Ah, looks like it couldn't
https://github.com/openzfs/zfs/blob/5236eafdcdf5e9e4e4d276f3c91391cdf5d97589/lib/libzfs/libzfs_crypto.c#L1285-L1291

Reply:
That would just be TOO CONVENIENT